Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -525,6 +532,7 @@ private bool EnsureOwningGrid()
                 if (this.OwningGrid != _owningGrid)
                 {
                     _owningGrid = this.OwningGrid;
+                    _owningGrid.Columns.CollectionChanged += new NotifyCollectionChangedEventHandler(Columns_CollectionChanged);
```

Comment:
There's no chance the event will be registered more than once here?

Reply:
Ah, you're right. I had copied existing code from the Checkbox column and it was wrong. Has been wrong for many years in Silverlight. 
The problem was that the unhooking only happened for the Remove action. The Replace and Reset cases were missed.  So I generalized the code by checking if the old OwningGrid is now null (in the latest commit).
Since a column has to leave a DataGrid before if joins another one, this code must be good now. Thanks Kevin.